src/react-app-env.d.ts

```diff
@@ -1 +1,41 @@
 /// <reference types="react-scripts" />
+
+import { Web3Provider } from '@ethersproject/providers'
+
+declare module 'jazzicon' {
+  export default function (diameter: number, seed: number): HTMLElement
+}
```

Comment:
What is this for?

Reply:
Hmm.. I'm not sure. Is it sth automatically generated code?